python_backend/services/base_service.py

```
@@ -0,0 +1,30 @@
+"""
+기본 서비스 클래스
+모든 서비스의 공통 기능을 제공하는 베이스 클래스
+"""
+
+import logging
+from abc import ABC
+
+
+class BaseService(ABC):
```

Ruff (0.12.2)

`BaseService` is an abstract base class, but it has no abstract methods or properties

(B024)